docs/user-guide/connectors/slack.rst

```diff
@@ -65,7 +71,7 @@ You need to supply a ``credentials.yml`` with the following content:
 
    slack:
      slack_token: "xoxb-286425452756-safjasdf7sl38KLls"
-     slack_channel: "#my_channel"
+     slack_channel: "C011GR5D33F"
```

Comment:
did #channel not work?

Reply:
I could not get it to work

Reply:
It will not respond to messages in the channel if you use the name. The [connector](https://github.com/RasaHQ/rasa/blob/a095714f5e59c393d91ffa45b6eda6ac620a27a6/rasa/core/channels/slack.py#L391) won't accept it based on the name. Slack is passing us the id.

```
Received message on unsupported channel: C011GR5D33Q
```

Reply:
Okay. I think a lot has changed since Will added features to this connector so I'm not so familiar with it anymore. Seems you know what's up though 👍